mod/admin.php

```diff
@@ -1850,13 +1850,13 @@ function __get_theme_admin_form(&$a, $theme) {
  * 
  * @param App $a
  */
-function admin_page_logs_post(&$a) {
-	if(x($_POST,"page_logs")) {
+function admin_page_logs_post(App &$a) {
+	if (x($_POST,"page_logs")) {
 		check_form_security_token_redirectOnErr('/admin/logs', 'admin_logs');
 
-		$logfile 	=	((x($_POST,'logfile'))		? notags(trim($_POST['logfile']))	: '');
-		$debugging	=	((x($_POST,'debugging'))	? true					: false);
-		$loglevel 	=	((x($_POST,'loglevel'))		? intval(trim($_POST['loglevel']))	: 0);
+		$logfile   = (x($_POST,'logfile'))    ? notags(trim($_POST['logfile']))	: '');
```

Comment:
Tab should be used at line start to indent code at correct position, only beatification (nicely arranging like this) should be done by space. Tabs may been shown as 4, 6, 8 spaces on different OS/editor.